controls/powershell.rb

```diff
@@ -22,6 +23,6 @@
   ref 'CIS Microsoft Windows Server 2012 R2 Benchmark', url: 'https://benchmarks.cisecurity.org/tools2/windows/CIS_Microsoft_Windows_Server_2012_R2_Benchmark_v2.2.1.pdf'
   describe registry_key('HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows\PowerShell\Transcription') do
     it { should exist }
-    its('EnableTranscripting') { should eq 0 }
```

Comment:
According to CIS this should be disabled. If we need an attribute to active it, lets add this